lesson02/main.go

```diff
@@ -3,13 +3,23 @@ package main
 import (
 	"fmt"
 
-	"github.com/SergeiLisitsyn/gocourse/fibonacci"
+	"github.com/SergeiLisitsyn/gocourse/lesson02/fibonacci"
 )
 
 func main() {
-	var f int = 12 // Quontity of Fibonacci numbers
-	var fiboArr []int32
-	fiboArr = fibonacci.Fibo(f)
-	fmt.Println(fiboArr)
+	defer fmt.Println("\nThe Progam has successfully completed!") //Complete massage
+	fmt.Println("Hello, please input f - qauntity of Fibonacci numbers you want to print")
+	var f int
+	fmt.Scanf("%d", &f)
+	fmt.Println("You entered:", f)
+	// f = 10 // Size of Fibonacci sequence
+	if f > 0 {
+		defer fmt.Print("\nNow you see sequence of ", f, " Fibonacci numbers.")
+	}
+	fibonacci.Fibo(f)
+	fmt.Println()
+	for k := 1; k <= f; k++ {
```

Comment:
You basically wrote the content of your Printer function here. You should wrap it into a separate Printer function and call it here.